Jenkinsfile

```diff
@@ -49,7 +49,7 @@ def gitMerge(String primaryBranch) {
   }
 }
 
-node("primary") {
+node("bedrock-macos") {
```

Comment:
Should we come up with another label here and apply it to the M1 nodes?

Reply:
I can, but I'm not sure if there's a process. I'll ask in Slack.